oxidation/libparsec/crates/platform_local_db/src/executor.rs

```diff
@@ -0,0 +1,89 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+use std::thread::JoinHandle;
+
+use diesel::SqliteConnection;
+use platform_async::channel;
+
+use crate::{DatabaseError, DatabaseResult};
+
+pub struct SqliteExecutor {
+    job_sender: Option<channel::Sender<ExecOrStop>>,
+    handle: Option<JoinHandle<()>>,
```

Comment:
Why do you need `Option` for both ?

Reply:
The option on `job_sender` isn't required, I forgot to remove it.
The option on `handle` is required to be able to `join` it in the `Drop` (the function `join` take the ownership of the value)

Reply:
You can `join` without `Option`